pkg/snowflake/role.go

```diff
@@ -1,8 +1,25 @@
 package snowflake
 
+import (
+	"database/sql"
+
+	"github.com/jmoiron/sqlx"
+)
+
 func Role(name string) *Builder {
 	return &Builder{
 		entityType: RoleType,
 		name:       name,
 	}
 }
+
+type role struct {
+	Name    sql.NullString `db:"name"`
+	Comment sql.NullString `db:"comment"`
+}
+
+func ScanRole(row *sqlx.Row) (*role, error) {
+	r := &role{}
+	e := row.StructScan(r)
+	return r, e
```

Comment:
```suggestion
	err := row.StructScan(r)
	return r, err
```
for consistency/convention